model/metrics/database-metrics.yaml

```diff
@@ -33,6 +33,154 @@ groups:
+            in which case it SHOULD be the first collection name in the query.
+      - ref: db.system
+        # TODO: Not adding to the minimal because of https://github.com/open-telemetry/build-tools/issues/192
+        requirement_level: required
+      - ref: network.peer.address
+        brief: Peer address of the database node where the operation was performed.
+        requirement_level:
+          recommended: If applicable for this database system.
+        note: >
+          Semantic conventions for individual database systems SHOULD document whether `network.peer.*` attributes are applicable.
+          Network peer address and port are useful when the application interacts with individual database nodes directly.
+
+          If a database operation involved multiple network calls (for example retries), the address of the last contacted node SHOULD be used.
+      - ref: network.peer.port
+        requirement_level:
+          recommended: If and only if `network.peer.address` is set.
+      - ref: db.namespace
+        requirement_level:
+          conditionally_required: If available.
+      - ref: thread.id
+        requirement_level:
+          conditionally_required: If available
+
+  - id: metric.db.client.operation.cpu_time
+    type: metric
+    metric_name: db.client.operation.cpu_time
+    brief: "CPU time spent for database client operations."
+    instrument: histogram
+    unit: "s"
+    stability: experimental
+    extends: attributes.db.client.minimal
+    attributes:
+      - ref: db.collection.name
+        requirement_level:
+          conditionally_required: >
+            If readily available. The collection name MAY be parsed from the query text,
+            in which case it SHOULD be the first collection name in the query.
+      - ref: db.system
+        # TODO: Not adding to the minimal because of https://github.com/open-telemetry/build-tools/issues/192
+        requirement_level: required
+      - ref: network.peer.address
+        brief: Peer address of the database node where the operation was performed.
+        requirement_level:
+          recommended: If applicable for this database system.
+        note: >
+          Semantic conventions for individual database systems SHOULD document whether `network.peer.*` attributes are applicable.
+          Network peer address and port are useful when the application interacts with individual database nodes directly.
+
+          If a database operation involved multiple network calls (for example retries), the address of the last contacted node SHOULD be used.
+      - ref: network.peer.port
+        requirement_level:
+          recommended: If and only if `network.peer.address` is set.
+      - ref: db.namespace
+        requirement_level:
+          conditionally_required: If available.
+      - ref: thread.id
+        requirement_level:
+          conditionally_required: If available
+
+  - id: metric.db.client.operation.pages_scanned
```

Comment:
since not all databases use pages, I don't think it makes sense to have this on the "general" view of the database. If there are things that only make sense for a specific group, is worth having those split from here.

Reply:
I definitely get the reasoning there and don't necessarily disagree, but if the concept of "pages" (or another example) is shared by multiple databases, either now or in the future, would it make more sense to have a common metric name for them or multiple db-specific names? And could it be more confusing to have some metrics have common names while others would fall under db-specific namespaces? Happy to move these metrics elsewhere if these topics have already been discussed/answered though.